### ISSUE TITLE
fused image: Save as TIFF stack calibration not saved

### ISSUE THREAD

**Issue by schmiedc** [bigdataviewer/SPIM_Registration#71]
Hi Stephan,

I currently use the save as TIFF stack as setting for the cluster.
However the metadata (calibration) is not preserved in the resulting .tif files.

Thanks a lot!
Christopher

**Comment by StephanPreibisch** [comment]
Hi Christopher, this is on purpose. The calibration of the final image represents the global coordinate system in which everything is registered, so any coordinate in any fused image - even over different bounding boxes - refers to the comparable position in 3d-space.

**Comment by schmiedc** [comment]
Hi Stephan, 

So do I understand correctly that you sacrifice giving the sizes in µm for comparability over different bounding boxes and downsampling? 

Cheers,
Christopher

**Comment by StephanPreibisch** [comment]
yes, but I could make it an option for export ...

**Comment by schmiedc** [comment]
Makes sense the global coordinate system is indispensable for the bounding box of the average fusion. 

Good, it would be great to have that option! Then I would not need to manually specify the calibration, which is another break point for the user.
